c/busybox-1.22.0/mkfifo-incomplete_true-no-overflow_true-valid-memsafety.c

```diff
@@ -210,16 +210,21 @@ static signed long int full_write(signed int fd, const void *buf, unsigned long
   return total;
 }
 
+extern unsigned int __VERIFIER_nondet_uint(void);
+
```

Comment:
I would prefer if that were added to busybox_sv_comp.h (even though it requires re-preprocessing all .c files).